modules/atomic-widgets/widgets/atomic-heading.php

```diff
@@ -26,6 +27,10 @@ public function get_icon() {
 		return 'eicon-t-letter';
 	}
 
+	public function has_widget_container(): bool {
+		return ! Plugin::$instance->experiments->is_feature_active( 'e_optimized_markup' );
+	}
+
```

Comment:
We can move this function to `Atomic_Widget_Base` and have it always return `false` (here and in the Atomic Image)